labs/bdls.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: default
+title: BDLS
+[//]: # (parent: Labs)
+---
+# Lab Name
+BDLS
+
+# Short Description
+BDLS is an innovative BFT consensus algorithm that features safety and liveness by presenting a mathematically proven secure BFT protocol that is resilient in open networks such as the Internet. BDLS overcomes many problems, such as DoS attacks, as well as the deadlock problem caused by unreliable p2p broadcast channels. These problems are all very relevant to existing realistic open network scenarios, and are the focus of extensive work in improving Internet security, but it is an area largely ignored by most in mainstream BFT protocol design. 
+(Paper: https://eprint.iacr.org/2019/1460.pdf)
+
+
+# Scope of Lab
+BDLS library is planned to be integrated into Hyperledger Fabric as its ordering service, eventually replacing Fabric's Raft ordering service. Other distributed ledger projects under Hyperledger can also benefit from and use BDLS.
```

Comment:
I'm sorry but I think this is too strong a wording at the very least. There is currently no such plans that have been agreed upon. We don't even have an RFC on adopting BDLS.
This isn't to say that this lab isn't interesting but we need to be careful about setting expectations.

Reply:
Sorry for the wording, please review the updated one.